MAUI/ListView/emptyview.md

```diff
@@ -0,0 +1,320 @@
+
+The `EmptyView` appearance can be changed at run time based on its value using `EmptyViewTemplate` property.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<ContentPage xmlns:syncfusion="clr-namespace:Syncfusion.Maui.ListView;assembly=Syncfusion.Maui.ListView"
+             xmlns:local="clr-namespace:EmptyViewTemplate">
+  <ContentPage.Resources>
+    <ResourceDictionary>          
+      <DataTemplate x:Key="BasicTemplate">
+        <Label Text="{Binding .,StringFormat='{0} is not found'}" 
+               HorizontalTextAlignment="Center" VerticalOptions="CenterAndExpand"
+               FontSize="18" FontFamily="Roboto-Regular"/>
+      </DataTemplate>
+      <DataTemplate  x:Key="AdvancedTemplate">
+        <StackLayout VerticalOptions="CenterAndExpand">
+          <Label Text="&#xe725;" 
+                 FontSize="40"
+                 FontFamily="{OnPlatform iOS=ListViewFontIcons, MacCatalyst=ListViewFontIcons, Android=ListViewFontIcons.ttf#, UWP=ListViewFontIcons.ttf#ListViewFontIcons}"
+                 HorizontalTextAlignment="Center"/>
+          <Label Text="{Binding .,StringFormat='{0} is not found'}" 
+                 FontSize="16" 
+                 FontFamily="Roboto-Regular" 
+                 HorizontalTextAlignment="Center"/>
+        </StackLayout>
+      </DataTemplate>
+      <local:EmptyViewDataTemplateSelector x:Key="DataTemplateSelector" 
+                                           BasicTemplate="{StaticResource BasicTemplate}" 
+                                           AdvancedTemplate="{StaticResource AdvancedTemplate}"/>
+    </ResourceDictionary>
+  </ContentPage.Resources> 
+
+  <Grid>
+    <Grid.RowDefinitions>
+      <RowDefinition Height="30"/>
+      <RowDefinition Height="*" />
+    </Grid.RowDefinitions>
+    <SearchBar x:Name="filterText" 
+               FontSize="16"                              
+               Placeholder="Filter Inventory" TextChanged="SearchBar_TextChanged"/>                                                   
+    <syncfusion:SfListView Grid.Row="1" x:Name="listView"
+                           ItemsSource="{Binding Items}"
+                           ItemSize="56"
+                           EmptyView="{Binding Source={x:Reference filterText},Path=Text}"
+                           EmptyViewTemplate="{StaticResource DataTemplateSelector}">                         
+    </syncfusion:SfListView>
+  </Grid>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+listView.EmptyView = filterText.Text;
+
+listView.EmptyViewTemplate = new EmptyViewDataTemplateSelector() { BasicTemplate = this.Resources["BasicTemplate"] as DataTemplate, AdvancedTemplate = this.Resources["AdvancedTemplate"] as DataTemplate };
+{% endhighlight %}
+{% endtabs %}
+
+The `SfListView.EmptyView` property is set to the SearchBar.Text property, and the `SfListView.EmptyViewTemplate` property is set to a EmptyViewDataTemplateSelector object.

```

Comment:
The `SfListView.EmptyView` property is set to the SearchBar.Text property, and the `SfListView.EmptyViewTemplate` property is set to an EmptyViewDataTemplateSelector object.